jdbc-h2/src/main/resources/migrations/h2/V1_25__dashboard.sql

```diff
@@ -1,5 +1,5 @@
 CREATE TABLE IF NOT EXISTS dashboards (
-                                     "key" VARCHAR(250) NOT NULL PRIMARY KEY,
```

Comment:
Please revert, this will fail existing installation as the file will be detected by Flyway as different (not 100% sure)